client/my-sites/activity/activity-log-item/index.jsx

```diff
@@ -255,6 +258,19 @@ class ActivityLogItem extends Component {
 						{ translate( 'Restore to this point' ) }
 					</PopoverMenuItem>
 
+					{ disableRestore && (
+						<PopoverMenuItem
+							icon="plus"
+							href={
+								canAutoconfigure
+									? `/start/rewind-auto-config/?blogid=${ siteId }&siteSlug=${ siteSlug }`
+									: `/settings/security/${ siteSlug }`
+							}
+						>
+							{ translate( 'Add server credentials in order to restore your site' ) }
```

Comment:
Can we change the copy to 
`Add server credentials to restore site` I think its more succinct.

Reply:
Now that you highlight it Enej, and make me pay more close attention, I feel both are misleading because it's not really "restoring the site", it's "allowing for restore to be possible", so maybe something like this would work?

> Add server credentials to enable restoring

Reply:
I like those suggestions, @enejb @folletto ! “Enable restoring” sounds very sensible to me, let's go with that for now.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/40014/commits/20316703f8b1f45d79be1a0270fcf0bb0ee060dc